api/v1alpha1/httproute_types.go

```diff
@@ -264,6 +264,45 @@ type ForwardToTarget struct {
 	//
 	// +optional
 	TargetPort *TargetPort `json:"targetPort" protobuf:"bytes,2,opt,name=targetPort"`
+
+	// Timeout specifies the maximum time to wait for a response after sending
+	// an HTTP request to targetRef.
+	//
+	// Support: Core
+	//
+	// +optional
+	Timeout *metav1.Duration `json:"timeout,omitempty" protobuf:"bytes,3,opt,name=timeout"`
+
+	// RetryPolicy describes a policy for resending a failed request to a targetRef.
```

Comment:
What defines a request as failed?
Will it be an L4 failure or L7 failure (idempotent HTTP methods)?

Reply:
I updated the docs. A connection is considered failed when the connection is either refused (5xx) or times out.t